### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by zhcf** [deric/mesos-torque#1]
Hello. I want to use the mesos as the basis of the hpc and bigdata cluster, the bigdata yarn can work on the mesos, but torque can not work on it. I see your code, I am curious whether this had ever worked? in the code, the executor just start pbs_mom, do nothing else.

**Comment by deric** [comment]
Hi, no not really. The files were copies from Mesos project, in the original paper Benjamin Hindman was running some MPI jobs on top of Mesos. I wanted to create a simple PBS like CLI interface for Mesos, but  I didn't have time to finish it.

**Comment by zhcf** [comment]
ok, do you have any idea about how to make the PBS CLI working on Mesos?

**Comment by deric** [comment]
Yeah, a server (a mesos framework) is needed which needs to negotiate resource allocations in the cluster and keep a tasks queue. Then the CLI would just communicate with this server. I guess I could write a simple proof of concept, of course not with so many features as PBS. Ideally the server would act also as a fileserver for mesos-agents which would be given an URI to fetch a binary.

**Comment by zhcf** [comment]
So you mean that the torque agent(pbs_mom) will not be used,  the torque CLI will communicate with the server(mesos framenwork), in this case ,all commands of torque need to be implemented again?

**Comment by deric** [comment]
Yeah, another option would be using PBS API and basically replacing [maui](https://github.com/LabAdvComp/maui) scheduler. I'm not really familiar with that API, but I guess that some hacks would be needed to make it work.
